patch-Disable-build-id.patch

```diff
@@ -0,0 +1,59 @@
+From 44b05af0e88ff9f1c7ccb1650b4318fc3df12554 Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20Pierret=20=28fepitre=29?=
+ <frederic.pierret@qubes-os.org>
+Date: Sun, 8 Nov 2020 10:42:31 +0100
+Subject: [PATCH] Disable build-id
+
+---
+ Config.mk             |  6 ------
+ xen/arch/x86/Makefile | 16 +---------------
+ 2 files changed, 1 insertion(+), 21 deletions(-)
+
+diff --git a/Config.mk b/Config.mk
+index ab8a232693..909936d5ef 100644
+--- a/Config.mk
++++ b/Config.mk
+@@ -129,13 +129,7 @@ ld-ver-build-id = $(shell $(1) --build-id 2>&1 | \
+ 					grep -q build-id && echo n || echo y)
+ 
+ export XEN_HAS_BUILD_ID ?= n
+-ifeq ($(call ld-ver-build-id,$(LD)),n)
+ build_id_linker :=
+-else
+-CFLAGS += -DBUILD_ID
+-export XEN_HAS_BUILD_ID=y
+-build_id_linker := --build-id=sha1
+-endif
+ 
+ ifndef XEN_HAS_CHECKPOLICY
+     CHECKPOLICY ?= checkpolicy
+diff --git a/xen/arch/x86/Makefile b/xen/arch/x86/Makefile
+index 9b368632fb..ad5f6dfa51 100644
+--- a/xen/arch/x86/Makefile
++++ b/xen/arch/x86/Makefile
+@@ -188,21 +188,7 @@ EFI_LDFLAGS += --major-subsystem-version=2 --minor-subsystem-version=0
+ $(TARGET).efi: VIRT_BASE = 0x$(shell $(NM) efi/relocs-dummy.o | sed -n 's, A VIRT_START$$,,p')
+ $(TARGET).efi: ALT_BASE = 0x$(shell $(NM) efi/relocs-dummy.o | sed -n 's, A ALT_START$$,,p')
+ 
+-ifneq ($(build_id_linker),)
+-ifeq ($(call ld-ver-build-id,$(LD) $(filter -m%,$(EFI_LDFLAGS))),y)
+-CFLAGS-y += -DBUILD_ID_EFI
+-EFI_LDFLAGS += $(build_id_linker)
+-note_file := efi/buildid.o
+-# NB: this must be the last input in the linker call, because inputs following
+-# the -b option will all be treated as being in the specified format.
+-note_file_option := -b pe-x86-64 $(note_file)
+-else
+-note_file := note.o
+-endif
+-else
+-note_file :=
+-endif
+-note_file_option ?= $(note_file)
++note_file_option ?=
```

Comment:
The above looks brutal but now I have a reproducible build so I can tune if needed.